cmds/shutdown/shutdown.go

```diff
@@ -47,17 +65,44 @@ func main() {
 		op = flag.Args()[0]
 	}
 
-	f, ok := opcodes[op]
+	if op == "" {
+		if flags.h && flags.r {
+			return 0, errors.New("Invalid aruments")
+		} else if flags.h {
+			op = "halt"
+		} else {
```

Comment:
so we reboot if no flag or op is given?

i'd have expected

```go
} else if flags.r {
  op = "reboot"
} else {
  return 0, fmt.Errorf("no operation given")
}
```

but also idk what a standard Linux shutdown command will do.

Reply:
Yea, I was aiming to add the flags without altering the existing functionality, which is to reboot by default.